code/renderers/svelte/src/public-types.ts

```diff
@@ -41,7 +41,7 @@ export type StoryFn<TCmpOrArgs = Args> = TCmpOrArgs extends SvelteComponentTyped
  * @see [Named Story exports](https://storybook.js.org/docs/formats/component-story-format/#named-story-exports)
  */
 export type StoryObj<MetaOrCmpOrArgs = Args> = MetaOrCmpOrArgs extends {
-  render?: ArgsStoryFn<SvelteRenderer, any>;
+  render?: ArgsStoryFn<any, any>;
```

Comment:
is this intentional? is SvelteRenderer not correct anymore?

Reply:
I can revert this yes. Was trying around with explicit events for Svelte.